backend/src/main/java/com/bakdata/conquery/apiv1/query/concept/filter/CQTable.java

```diff
@@ -88,4 +92,11 @@ public ValidityDate findValidityDate() {
 		return null;
 	}
 
+	public boolean hasSelectedSecondaryId(SecondaryIdDescription secondaryId) {
```

Comment:
Das hier habe ich auch nur aus dem CQConcept in eine eigene Method heraus gezogen. So kann ich das auch leicht für die Conversion verwenden.